userspace/libscap/scap_bpf.c

```diff
@@ -450,8 +464,8 @@ static int32_t load_tracepoint(scap_t* handle, const char *event, struct bpf_ins
 	fd = bpf_load_program(prog, program_type, insns_cnt, error, BPF_LOG_SIZE);
 	if(fd < 0)
 	{
-		fprintf(stderr, "%s", error);
-		snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "bpf_load_program() err=%d event=%s message=%s", errno, event, error);
+		fprintf(stderr, "-- BEGIN PROG LOAD LOG --\n%s-- END PROG LOAD LOG --\n", error);
```

Comment:
This is a nitpick, but maybe we should place the separator on a new line to improve readability? Or are we relying on the verifier always adding a new line to the end of the errors?
```suggestion
		fprintf(stderr, "-- BEGIN PROG LOAD LOG --\n%s\n-- END PROG LOAD LOG --\n", error);
```

Reply:
Thank you Mauro I took it directly from `libbpf` code so it should be ok without `\n`, but just to be sure I see no problem in adding it :)